packages/node/__tests__/lib/process-request.test.js

```diff
@@ -43,9 +45,9 @@ describe('processRequest()', () => {
 
         return request(app)
           .post('/')
-          .send({ a: { b: { c: 1 } } })
+          .send({ a: { b: { c: {} } } })
```

Comment:
Can this still handle redacting `c` if it's `1`?

Reply:
yup, I just made it an object to tests non-primitives—ended up not really being consequential tho.

haha aka I didn't just pass the test by changing the input value 🙈